templates/case/_delete.html.twig

```diff
@@ -0,0 +1,23 @@
+{% trans_default_domain 'case' %}
+
+<div class="modal fade" id="delete_case">
+    <div class="modal-dialog">
+        <div class="modal-content">
+            <div class="modal-header">
+                <h5 class="modal-title" id="assignCaseworkerModalLabel">{% trans %}Delete case{% endtrans %}</h5>
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                    <span aria-hidden="true"><i class="fas fa-times"></i></span>
+                </button>
+            </div>
+            <div class="modal-body">
+                {{ form_start(delete_form, {'action': path('case_delete', {'id': case.id})}) }}
+                {{ form_widget(delete_form) }}
+            </div>
+            <div class="modal-footer">
+                <button type="button" class="btn btn-secondary" data-dismiss="modal">{% trans %}Cancel{% endtrans %}</button>
+                <button type="submit" class="btn btn-danger">{% trans %}Confirm deletion{% endtrans %}</button>
+                {{ form_end(delete_form) }}
+            </div>
```

Comment:
The start and end tags generated here are not balanced:

```html
<div …>
  <form …>
  …
</div>
<div …>
  …
  </form>
</div>
```

The `form` element must contain both `div .modal-body` and `div.modal-footer` to work as expected.

Reply:
Made sure `form` element contains both `modal-body` and `modal-footer`.